implementations/rust/ockam/ockam_examples/example_projects/no_std/Cargo.toml

```diff
@@ -33,22 +33,30 @@ cortexm = [
     "cortex-m",
     "cortex-m-rt",
     "cortex-m-semihosting",
-    "panic-semihosting"
+    "panic-semihosting",
+    "log-semihosting",
+    "tracing",
+    "ockam_core/no_std"
 ]
 
 qemu = ["no_std", "alloc", "cortexm"]
 atsame54 = ["no_std", "alloc", "cortexm", "atsame54_xpro"]
 stm32f4 = ["no_std", "alloc", "cortexm", "stm32f4xx-hal"]
 
+log-itm = []
+log-semihosting = []
+log-uart = []
 
 [dependencies]
 ockam = { path = "../../../ockam", default_features = false, features = ["software_vault"] }
+ockam_core = { path = "../../../ockam_core", default_features = false, optional = true }
```

Comment:
Can we just reexport things you need from ockam_core to ockam? I think we want to avoid situations when you need to import other ockam crates that are already a dependency for `ockam` crate itself. Isn't it?

Reply:
Ah, nice catch - I switched the example over to use `tracing` for output instead of `println` - completely forgot to remove the dependency.